compute/quickstart/compute_quickstart_sample.go

```diff
@@ -0,0 +1,110 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+var (
+	client     *mongo.Client
+	collection *mongo.Collection
+	ctx        = context.Background()
+	mongohost  string
+)
+
+func main() {
+	var err error
+	mongohost = os.Getenv("HOST")
+	uri := fmt.Sprintf("mongodb://%s:%s", mongohost, "80")
+	clientOptions := options.Client().ApplyURI(uri)
+
+	if client, err = mongo.Connect(ctx, clientOptions); err != nil {
+		log.Fatal(fmt.Printf("error connecting to mongo: %s", err))
+	}
+
+	collection = client.Database("test").Collection("trainers")
+
+	trainers := []interface{}{
+		trainer{"Ash", 10, "Pallet Town"},
+		trainer{"Misty", 10, "Cerulean City"},
+		trainer{"Brock", 15, "Pewter City"},
+	}
+
+	if _, err := collection.InsertMany(ctx, trainers); err != nil {
+		log.Fatal(fmt.Printf("error inserting records to mongo: %s", err))
+	}
+
+	http.HandleFunc("/", listHandler)
+	log.Fatal(http.ListenAndServe(":80", nil))
+}
+
+func listHandler(w http.ResponseWriter, r *http.Request) {
+	trainers, err := listTrainers()
+	if err != nil {
+		w.WriteHeader(http.StatusInternalServerError)
+		w.Write([]byte(err.Error()))
+		return
+	}
+
+	j, err := json.Marshal(trainers)
+	if err != nil {
+		w.WriteHeader(http.StatusInternalServerError)
+		w.Write([]byte(err.Error()))
+		return
+	}
+
+	w.WriteHeader(http.StatusOK)
+	w.Write(j)
+	return
+}
+
+type trainer struct {
+	Name string
+	Age  int
+	City string
+}
+
+func listTrainers() ([]*trainer, error) {
```

Comment:
A GoDoc here would be good: https://github.com/GoogleCloudPlatform/golang-samples/blob/main/CONTRIBUTING.md#comment-functions-and-packages.